src/main/kotlin/com/cognifide/gradle/aem/AemPlugin.kt

```diff
@@ -76,6 +77,9 @@ class AemPlugin : Plugin<Project> {
         val upload = project.tasks.create(UploadTask.NAME, UploadTask::class.java)
         val install = project.tasks.create(InstallTask.NAME, InstallTask::class.java)
         val activate = project.tasks.create(ActivateTask.NAME, ActivateTask::class.java)
+        val uninstall = project.tasks.create(UninstallTask.NAME, UninstallTask::class.java)
```

Comment:
how about creating tasks here: https://github.com/Cognifide/gradle-aem-plugin/blob/444fbfb575e1ae897c22458fd6c066db352bdc84/src/main/kotlin/com/cognifide/gradle/aem/AemPlugin.kt#L116

with not introducing variables for them to avoid using such annotation? it could hide another really unused declarations in the future